src/third-party/PhishingController.ts

```diff
@@ -133,30 +135,40 @@ export class PhishingController extends BaseController<
     };
     this.detector = new PhishingDetector(this.defaultState.phishing);
     this.initialize();
-    this.poll();
   }
 
   /**
-   * Starts a new polling interval.
+   * Set the interval at which the phishing list will be refetched. Fetching will only occur on the next call to test/bypass. For immediate update to the phishing list, call updatePhishingLists directly.
    *
-   * @param interval - Polling interval used to fetch new approval lists.
+   * @param interval - the new interval, in ms
```

Comment:
Interesting that our linter is not flagging this as a violation as we usually end descriptions for parameters with periods 🤔 . I'll make an issue for this.

Reply:
I have to rebase, so ill fix this up, nice catcherino!